vector/src/main/java/im/vector/app/features/notifications/NotifiableEventResolver.kt

```diff
@@ -192,6 +199,26 @@ class NotifiableEventResolver @Inject constructor(
         }
     }
 
+    private suspend fun TimelineEvent.fetchImageIfPresent(session: Session): Uri? {
+        return when {
+            root.isEncrypted() && root.mxDecryptionResult == null -> null
+            root.getClearType() == EventType.MESSAGE              -> downloadAndExportImage(session)
```

Comment:
I think we should filter on `msgtype` here, since the attachment is not necessarily an image, be can be anything else. `Event.isImageMessage()` can help to do that.

Reply:
:+1:  will do!